mautrix_hangouts/user.py

```diff
@@ -277,10 +296,12 @@ async def on_connect_later(self) -> None:
     async def on_reconnect(self) -> None:
         self.connected = True
         await self.send_bridge_notice("Reconnected to Hangouts")
+        await self.push_bridge_state(BridgeStateEvent.CONNECTED)
 
     async def on_disconnect(self) -> None:
         self.connected = False
         await self.send_bridge_notice("Disconnected from Hangouts")
+        await self.push_bridge_state(BridgeStateEvent.UNKNOWN_ERROR, error="hangouts-disconnected")
```

Comment:
These are transient disconnects, only the listener returning stops retrying.

```suggestion
        await self.push_bridge_state(BridgeStateEvent.TRANSIENT_DISCONNECT, error="hangouts-disconnected")
```